src/PollingBlockTracker.test.ts

```diff
@@ -834,6 +834,262 @@ describe('PollingBlockTracker', () => {
         },
       );
     });
+
+    describe('after a block number is cached', () => {
+      it('should return the fetched block number if the fetched blocked number is greater than the current block number', async () => {
```

Comment:
Nit: `fetched blocked number` -> `fetched block number` (there are multiple instances of this)

Reply:
that's embarrassing. good catch!